include/gmapping/gridfastslam/gridslamprocessor.h

```diff
@@ -305,11 +308,10 @@ namespace GMapping {
     // stream in which to write the messages
     std::ostream& m_infoStream;
     
-    
     // the functions below performs side effect on the internal structure,
     //should be called only inside the processScan method
   private:
-    
+    std::vector<double> durations;
```

Comment:
because I want to store the execution time of each function call. so it has to be valid outside the function's scope as well.